app/uk/gov/hmrc/nisp/models/NIRecordTaxYear.scala

```diff
@@ -16,14 +16,33 @@
 
 package uk.gov.hmrc.nisp.models
 
+import org.joda.time.LocalDate
 import play.api.libs.json.Json
 import uk.gov.hmrc.nisp.utils.Constants
 
 case class NIRecordTaxYear(taxYear: Int, qualifying: Boolean,classOneContributions: BigDecimal,
                            classTwoCredits: Int, classThreeCredits: Int, otherCredits: Int,
                            classThreePayable: Option[BigDecimal], classThreePayableBy: Option[NpsDate],
                            classThreePayableByPenalty: Option[NpsDate], payable: Boolean, underInvestigation: Boolean) {
+
   val displayableTaxYear: String = s"$taxYear-${(taxYear + 1).toString.substring(Constants.shortYearStartCharacter,Constants.shortYearEndCharacter)}"
+
+}
+
+
+trait isCutOffDate {
```

Comment:
we do not need separate trait. we can just do it like we did for displayableTaxYear above.